selfdrive/car/fw_versions.py

```diff
@@ -209,7 +209,7 @@ def match_fw_to_car_exact(fw_versions_dict):
         continue
 
       # On some Toyota models, the engine can show on two different addresses
-      if ecu_type == Ecu.engine and candidate in [TOYOTA.CAMRY, TOYOTA.COROLLA_TSS2, TOYOTA.CHR, TOYOTA.LEXUS_IS, TOYOTA.AVALON] and found_version is None:
+      if ecu_type == Ecu.engine and candidate in [TOYOTA.CAMRY, TOYOTA.COROLLA_TSS2, TOYOTA.CHR, TOYOTA.LEXUS_IS, TOYOTA.AVALON_2019] and found_version is None:
```

Comment:
@adeebshihadeh Can we remove this? So far I've seen the engine on 7e0 for AVALON and 700 for AVALON_2019, but not both for either. It was probably added when the 19+ years were added